lib/api/v3/utilities/path_helper.rb

```diff
@@ -70,6 +70,10 @@ def self.priorities
             "#{root}/priorities"
           end
 
+          def self.priority(id)
+            "#{priorities}/#{id}"
```

Comment:
This is untested code :scream:

-> spec/lib/api/v3/utilities/path_helper_spec.rb